easybuild/easyconfigs/q/Qt/Qt-4.8.7-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,27 @@
+name = 'Qt'
+version = '4.8.7'
+shortversion = '4.8'
+# https://github.com/hpcugent/easybuild-easyconfigs/issues/2777
+#gcc434:ko version = '4.8.7'
+
+homepage = 'http://qt-project.org/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+
+source_urls = [ 'http://download.qt.io/official_releases/qt/%(shortversion)s/%(version)s/' ]
```

Comment:
The filename is `qt-everywhere-opensource-src-%(version)s.tar.gz`:
`http://download.qt.io/official_releases/qt/4.8/4.8.7/qt-everywhere-opensource-src-4.8.7.tar.gz`

Reply:
So what's wrong ?
> sources = [ 'qt-everywhere-opensource-src-%(version)s.tar.gz' ]

Reply:
It's a problem of the compute nodes only, as the download works on the login nodes instead.
I could reproduce the issue on Dom:
```
dom101:~$ srun -C gpu --pty /bin/bash
nid00002:~/production$ eb Qt-4.8.7-CrayGNU-17.08.eb
== temporary log file in case of crash /dev/shm/lucamar/easybuild/stage/tmp/eb-vO8kAy/easybuild-F3OOx5.log
== processing EasyBuild easyconfig /users/lucamar/production/easybuild/easyconfigs/q/Qt/Qt-4.8.7-CrayGNU-17.08.eb
== building and installing Qt/4.8.7-CrayGNU-17.08...
== fetching files...
== FAILED: Installation ended unsuccessfully (build directory: /dev/shm/lucamar/easybuild/stage/build/Qt/4.8.7/CrayGNU-17.08): build failed (first 300 chars): Couldn't find file qt-everywhere-opensource-src-4.8.7.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /users/lucamar/production/easybuild/easyconfigs/q/Qt/q/Qt/qt-everywhere-opensource-src-4.8.7.tar.gz, /users/lucamar/production/easybuild/easyconfigs/q/Qt/Qt/qt-
== Results of the build can be found in the log file(s) /dev/shm/lucamar/easybuild/stage/tmp/eb-vO8kAy/easybuild-Qt-4.8.7-20180202.134627.MlOAt.log
ERROR: Build of /users/lucamar/production/easybuild/easyconfigs/q/Qt/Qt-4.8.7-CrayGNU-17.08.eb failed (err: "build failed (first 300 chars): Couldn't find file qt-everywhere-opensource-src-4.8.7.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /users/lucamar/production/easybuild/easyconfigs/q/Qt/q/Qt/qt-everywhere-opensource-src-4.8.7.tar.gz, /users/lucamar/production/easybuild/easyconfigs/q/Qt/Qt/qt-")
```

Reply:
Manual download with `wget` works on the compute nodes of Dom:
dom101:~$ srun -C gpu --pty /bin/bash
nid00000:~$ wget http://download.qt.io/official_releases/qt/4.8/4.8.7/qt-everywhere-opensource-src-4.8.7.tar.gz
--2018-02-02 14:20:01--  http://download.qt.io/official_releases/qt/4.8/4.8.7/qt-everywhere-opensource-src-4.8.7.tar.gz
Resolving download.qt.io (download.qt.io)... 77.86.229.90
Connecting to download.qt.io (download.qt.io)|77.86.229.90|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: http://ftp.fau.de/qtproject/official_releases/qt/4.8/4.8.7/qt-everywhere-opensource-src-4.8.7.tar.gz [following]
--2018-02-02 14:20:01--  http://ftp.fau.de/qtproject/official_releases/qt/4.8/4.8.7/qt-everywhere-opensource-src-4.8.7.tar.gz
Resolving ftp.fau.de (ftp.fau.de)... 131.188.12.211, 2001:638:a000:1021:21::1
Connecting to ftp.fau.de (ftp.fau.de)|131.188.12.211|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 241075567 (230M) [application/x-gzip]
Saving to: ‘qt-everywhere-opensource-src-4.8.7.tar.gz’

100%[============================================================================================================================>] 241,075,567 15.0MB/s   in 15s    

2018-02-02 14:20:17 (14.9 MB/s) - ‘qt-everywhere-opensource-src-4.8.7.tar.gz’ saved [241075567/241075567]
```

Reply:
I now get occasionally the download error on the login nodes as well...